### ISSUE TITLE
fails to build with local numpy

### ISSUE THREAD

**Issue by MaxGaukler** [python-control/Slycot#65]
The CMake file only looks in the main python site directory, not in the other directories. Therefore, it fails to find a custom numpy installation via `pip`, which is however necessary if some other library requires a more recent version of numpy than available from the distribution's (here Ubuntu) packages.

See the following Dockerfile, which should work without the `ln -s` line, but doesn't.
```
FROM ubuntu:18.04
RUN apt-get -qy update && apt-get -qy install python3-pip
RUN apt-get -qy install cmake gfortran libopenblas-dev 
RUN pip3 install numpy scikit-build
# BUG: doesn't work without this line:
RUN ln -s /usr/local/lib/python3.6/dist-packages/numpy /usr/lib/python3/dist-packages/numpy  
RUN pip3 install slycot
```

The correct path to look for numpy (required for the f2py headers) can be determined with:
`python3 -c "from os.path import dirname; import numpy; print(dirname(dirname(numpy.__file__)))"`

**Comment by roryyorke** [comment]
@jakevdp [proposed this](https://github.com/python-control/Slycot/pull/66#issuecomment-503242165)

```
$ !sed -i "s/from distutils.sysconfig import get_python_lib as pl; print(pl())/import os,numpy; print(os.path.dirname(numpy.__path__[0]))/" Slycot/CMakeLists.txt
```

**Comment by roryyorke** [comment]
This is fixed in #67.  Hopefully soon available in 0.3.5 (#70).

Please reopen if the bug is still there on 0.3.5.
